Holovibes/mainwindow.ui

```diff
@@ -1094,6 +1124,9 @@
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="Q_Label">
+             <property name="toolTip">
+              <string>The SVD filter's start frequency</string>
```

Comment:
The SVD/SSA filter's start frequency : the range of frequencies kept is [z2, z2 + width].